util/reggen/reg_pkg.sv.tpl

```diff
@@ -18,6 +18,9 @@ package ${block.name}_reg_pkg;
   parameter ${param["type"]} ${param["name"]} = ${param["default"]};
 % endfor
 
+  // Address width within the block
+  parameter int BlockAW = ${block.addr_width};
```

Comment:
We went mostly for a camel-case variant in OpenTitan which would make this "BlockAw". Can you rename it?